Source/ACE.Server/Network/GameEvent/Events/GameEventFellowshipFullUpdate.cs

```diff
@@ -48,9 +37,14 @@ public GameEventFellowshipFullUpdate(Session session)
 
             Writer.Write(Convert.ToUInt32(fellowship.IsLocked));
 
-            // TODO PackableHashTable of fellows departed - fellowsDeparted  -<ObjectID,int>
-            Writer.Write((uint)0x00200000);
-            Writer.Write((uint)0x00200000);
+            PackableHashTable.WriteHeader(Writer, fellowship.DepartedMembers.Count);
```

Comment:
This is likely incorrect header. Should be using the example above from the fellowship hash table.

# of departed fellows, and what is likely a static bucket size (16?)